lib/typesense/helper.rb

```diff
@@ -34,6 +34,15 @@ def create
           type: 'geopoint',
           facet: false,
           optional: true
+        }, {
+          name: 'name',
```

Comment:
We may want to make this optional. All records should have a `name` attribute, but in the event that one doesn't (due to an import error or something) we don't want to prevent the project from being indexed.